{
  "contest": 232,
  "handle": "pipoca",
  "risk": "3",
  "title": "resolve() function might be susceptible to a denial-of-service (DoS) attack",
  "issueId": 34,
  "issueUrl": "https://github.com/code-423n4/2023-04-ens-findings/issues/34"
}